index.bs

```diff
@@ -416,11 +416,11 @@ The static {{DeviceMotionEvent/requestPermission()}} operation, when invoked, mu
 
 In the {{DeviceMotionEvent}} events fired by the user agent, the following requirements must apply:
 
-The {{DeviceMotionEvent/acceleration}} attribute must be initialized with the acceleration of the hosting device relative to the Earth frame, expressed in the body frame, as defined in [[#deviceorientation|deviceorientation Event]] section. The acceleration must be expressed in meters per second squared (m/s2).
+The {{DeviceMotionEvent/acceleration}} attribute must be initialized with the acceleration of the hosting device relative to the Earth frame, expressed in the body frame, as defined in [[#deviceorientation|deviceorientation Event]] section. The acceleration must be expressed in meters per second squared (m/s2) and must not be more precise than 0.1 m/s2.
```

Comment:
`m/s^2`?

Reply:
Thanks for the catch. 

HTML can represent superscript and this is in prose rather than code, so I've updated to use `<sup>2</sup>` in all the relevant places, I believe.